about.html

```diff
@@ -10,56 +10,77 @@
   <link rel="stylesheet" href="css/index.css">
 
 </head>
-
+<section>
 <body>
     <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+      <header>
+        <div class="headLinks">
+        
```

Comment:
You can also wrap the image and all the anchors in a `<nav>` element for navigation.